src/assets/Miguel3456.js

```diff
@@ -0,0 +1,8 @@
+export const Miguel3456 = {
+    "name": "Miguel",
+    "gif": "https://knowyourmeme.com/memes/woman-yelling-at-a-cat", 
```

Comment:
https://media2.giphy.com/media/JRE3AvLsSRXg360F6l/giphy.gif 

This is the one that gives the complete gif